lib/ui/home.dart

```diff
@@ -162,12 +162,12 @@ class Home extends StatelessWidget {
   _buildDiscover() {
     return ArticleList(
       style: ArticleListStyles.buildForDiscover(),
-      viewModelProvider: ArticleListProvider(
+      /*viewModelProvider: ArticleListProvider(
```

Comment:
All we have to do is n to provide a default. and it will look for it via the new injection system.